src/data/faq.json

```diff
@@ -2063,12 +2073,40 @@
                     "description": "See DCC"
                 }
             ],
+            "P": [
+            {
+                "term": "Proof of vaccination",
+                "description": "The proof of vaccination documents a vaccination procedure with a vaccine recommended by the Paul-Ehrlich-Institut. A vaccination proof must contain at least the following attributes: <ul><li>the personal data of the vaccinated person (at least surname, first name and date of birth)</li><li>date of vaccination, number of vaccinations,</li><li>designation of the vaccine,</li><li>name of the disease against which the vaccination was carried out, and</li><li>characteristics that indicate the person or institution responsible for carrying out the vaccination or issuing the proof of vaccination, for example, an official symbol or the name of the issuer. </li></ul> <br/><br/> For entry, according to § 2 number 10 Coronavirus-Einreiseverordnung, the vaccination certificate must still meet additional requirements (regarding languages and vaccination schema). <br/><br/> The proof of vaccination can be documented in different ways, one form is the digital => vaccination certificate."
+            }
+            ],
+            "R": [
+                {
+                    "term": "Recovery Certificate",
+                    "description": "The certificate of recovery is a => digital certificate issued to a person who has demonstrably recovered from COVID-19 and proved immunity by a PCR test."
+                }
+            ],
             "S": [
                 {
                     "term": "Signature",
                     "description": "A digital signature is a signature that is created by a crytographic function from a digital document. Digital signatures provide the following properties: <ul><li>Non-repudiation: The issuer cannot retrospectively deny signing the document.</li><li>Integrity: Altering the document invalidates the signature.</li><li>Authenticity: It is clear who signed the document.</li></ul> <br/><br/> As a consequence, the signature must fulfill the following requirements: <br/><ul><li>The signature must be computationally verifiable </li><li>Verifying the signature must be easy - forging a valid signature must be hard</li><li>The signature is only valid for the signed document and no other documents </li><li>The issuer must not be able to deny signing the document.</li></ul>"
                 }
-            ]
+            ],
+            "T": [
+                {
+                    "term": "Test certificate",
+                    "description": "A test est certificate is a => digital certificate issued to a person who has completed a COVID-19 test with a negative result."

```

Comment:
"est" should be removed.

Reply:
Done in https://github.com/corona-warn-app/cwa-website/pull/1543/commits/ca839c91bc7239d06157b7605f629f25985c48e0.